deeppavlov/configs/pipeline_manager/linear_classification.json

```diff
@@ -0,0 +1,178 @@
+          "class_name": "sklearn_component",
+          "component_name": "count",
+          "id": "count",
+          "save_path": "{DOWNLOADS_PATH}/vocabs/count.pkl",
+          "load_path": "{DOWNLOADS_PATH}/vocabs/count.pkl",
+          "model_class": "sklearn.feature_extraction.text:CountVectorizer",
+          "infer_method": "transform",
+          "lowercase": true,
+          "analyzer": "char",
+          "ngram_range": [1, 5],
+          "max_features": 10000,
+          "norm": null
+        }
+      ],
+      [
+        {
+          "in": ["x_vec"],
+          "out": ["y_pred_probas"],
+          "fit_on": ["x_vec", "y_ids"],
+          "class_name": "sklearn_component",
+          "component_name": "Log Reg",
+          "main": true,
+          "save_path": "{DOWNLOADS_PATH}/linm/logreg.pkl",
+          "load_path": "{DOWNLOADS_PATH}/linm/logreg.pkl",
+          "model_class": "sklearn.linear_model:LogisticRegression",
+          "infer_method": "predict_proba",
+          "warm_start": false,
+          "penalty": "l1",
+          "dual": false,
+          "C": {"random_range": [0.01, 2.0]},
+          "fit_intercept": {"random_bool": true},
+          "class_weight": {"random_choice": [null, "balanced"]},
+          "solver": {"random_choice": ["liblinear", "saga"]}
+        },
+        {
+          "in": ["x_vec"],
+          "out": ["y_pred_probas"],
+          "fit_on": ["x_vec", "y_ids"],
+          "class_name": "sklearn_component",
+          "component_name": "Log Reg",
+          "main": true,
+          "save_path": "{DOWNLOADS_PATH}/linm/logreg.pkl",
+          "load_path": "{DOWNLOADS_PATH}/linm/logreg.pkl",
+          "model_class": "sklearn.linear_model:LogisticRegression",
+          "infer_method": "predict_proba",
+          "warm_start": false,
+          "penalty": "l2",
+          "dual": false,
+          "C": {"random_range": [0.01, 2.0]},
+          "fit_intercept": {"random_bool": true},
+          "class_weight": {"random_choice": [null, "balanced"]},
+          "solver": {"random_choice": ["lbfgs", "newton-cg"]}
+        }
+      ],
+      [
+        {
+          "in": ["y_pred_probas"],
+          "out": ["y_pred_ids"],
+          "class_name": "proba2labels",
+          "component_name": "prob2ind",
+          "max_proba": true
+        }
+      ],
+      [
+        {
+          "in": ["y_pred_ids"],
+          "out": ["y_pred_labels"],
+          "component_name": "vocab",
+          "ref": "classes_vocab"
+        }
+      ]
+    ],
+    "out": ["y_pred_labels"]
+  },
+  "train": {
+    "metrics": [
+      {
+        "name": "f1_macro",
+        "inputs": ["y", "y_pred_labels"]
+      },
+      {
+        "name": "sets_accuracy",
+        "inputs": ["y", "y_pred_labels"]
+      },
+      {
+        "name": "f1_weighted",
+        "inputs": ["y", "y_pred_labels"]
+      }
+    ],
+    "show_examples": false,
+    "validate_best": true,
+    "test_best": false
+  },
+  "enumerate": {
```

Comment:
`enumerate` in one place, `pipeline_search` in others

Reply:
Thanks.